lbplanner/services/user/get_all_users.php

```diff
@@ -80,7 +80,7 @@ public static function get_all_users_returns(): external_multiple_structure {
         return new external_multiple_structure(
             new external_single_structure(
                 array(
-                    'userid' => new external_value(PARAM_INT, 'The id of the user'),
+                    'userid' => new external_value(PARAM_INT, 'The Moodle id of the user'),
```

Comment:
again:

do we have our own id's for users? if not, annotating the ID as 'Moodle id' is unnecessary and could cause confusion